ansible/group_vars/develop/cfg.yml

```
@@ -2,3 +2,6 @@ FEATURE_MATRIX_MESSENGER_ENABLED: "false"
 FEATURE_INSIGHTS_ENABLED: "false"
 FEATURE_NEXBOARD_ENABLED: "false"
 ROOM_VIEW_ENABLED: "true"
+FEATURE_IDENTITY_MANAGEMENT_ENABLED: "true"
+IDENTITY_MANAGEMENT__TENANT: "dbc"
+IDENTITY_MANAGEMENT__CLIENTID: "dbc"
```

if it's posible for both to go to the namespace as vaule?
```suggestion
IDENTITY_MANAGEMENT__TENANT: "{NAMESPACE}"
IDENTITY_MANAGEMENT__CLIENTID: "{NAMESPACE}"
```

---

The characters supported for TENANT or CLIENTID are not well documented. The characters [a-zA-Z0-9] are OK. For instance, [+*#?] are invalid. Is NAMESPACE limited somehow? Can we assume no 'special characters' here?

---

For how the namespace are build see https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/.github/workflows/deploy.yml#L90
Alternativ you cann posible use SC_THEME for it.
https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/group_vars/default/instance_cfg.yml

---

Thanks for the hint! SC_THEME seems a good option here.